pkg/estimator/server/server.go

```diff
@@ -137,6 +137,7 @@ func (es *AccurateSchedulerEstimatorServer) MaxAvailableReplicas(ctx context.Con
 	md, ok := metadata.FromIncomingContext(ctx)
 	if !ok {
 		klog.Warningf("No metadata from context.")
+		return nil, fmt.Errorf("no metadata from context")
```

Comment:
It's just a warning, do not return error.

Reply:
if !ok, md will be nil and the following calling `md.Get` will panic?

Reply:
done.